paired-fastq-to-unmapped-bam-fc.wdl

```diff
@@ -0,0 +1,145 @@
+## Copyright Broad Institute, 2018
+## 
+## This WDL converts paired FASTQ to uBAM and adds read group information 
+##
+## Requirements/expectations :
+## - Pair-end sequencing data in FASTQ format (one file per orientation)
+## - One or more read groups, one per pair of FASTQ files  
+## - A readgroup.list file with the following format :  
+##   ``readgroup   fastq_pair1   fastq_pair2   sample_name   library_name   platform_unit   run_date   platform_name   sequecing_center``
+##
+## Outputs :
+## - Set of unmapped BAMs, one per read group
+## - File of a list of the generated unmapped BAMs
+##
+## Cromwell version support 
+## - Successfully tested on v32
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+## For program versions, see docker containers. 
+##
+## LICENSING : 
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in 
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may 
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker 
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+# WORKFLOW DEFINITION
+workflow ConvertPairedFastQsToUnmappedBamWf {
+  File readgroup_list  
+  Array[Array[String]] readgroup_array = read_tsv(readgroup_list) 
+  String ubam_list_name = basename(readgroup_list,".list") + "unmapped.bam.list"
+  
+  String? gatk_docker_override
+  String gatk_docker = select_first([gatk_docker_override, "broadinstitute/gatk:latest"])
+  String? gatk_path_override
+  String gatk_path = select_first([gatk_path_override, "/gatk/gatk"])
+  Int? preemptible_attempts
+
+  # Convert multiple pairs of input fastqs in parallel
+  scatter (i in range(length(readgroup_array))) {
+
+    # Convert pair of FASTQs to uBAM
+    call PairedFastQsToUnmappedBAM {
+      input:
+      fastq_1 = readgroup_array[i][1],
```

Comment:
sorry this is a small thing, but it would be nice to indent this list of inputs :)

Reply:
done